modules/tinymce/src/core/main/ts/api/html/Styles.ts

```diff
@@ -39,6 +41,7 @@ const Styles = (settings: StylesSettings = {}, schema?: Schema): Styles => {
   /* jshint maxlen:255 */
   /* eslint max-len:0 */
   const urlOrStrRegExp = /(?:url(?:(?:\(\s*\"([^\"]+)\"\s*\))|(?:\(\s*\'([^\']+)\'\s*\))|(?:\(\s*([^)\s]+)\s*\))))|(?:\'([^\']+)\')|(?:\"([^\"]+)\")/gi;
+  const rgbRegExp = /rgb\s*\(\s*([0-9]+)\s*,\s*([0-9]+)\s*,\s*([0-9]+)\s*\)/gi;
```

Comment:
You missed `rgba`, and this already exists:
https://github.com/tinymce/tinymce/blob/f61d071878146aee69f57bdb712c1a6d0e8c8def/modules/acid/src/main/ts/ephox/acid/api/colour/RgbaColour.ts#L10-L11

In fact it's probably better to use `RgbaColour.fromString` the same way the `Transformations` code does.

Reply:
Ahh I see. I've updated it to now use the methods from Transformations instead of adding that regex